fiscal_epos_print/static/src/js/chrome.js

```diff
@@ -164,6 +167,23 @@ odoo.define("fiscal_epos_print.chrome", function (require) {
             });
         },
 
+        close_and_report: function(event) {
+            this.hide();
+            var printer_options = this.getPrinterOptions();
+            var fp90 = new eposDriver(printer_options, this);
+
+            var self = this;
+            this.gui.show_popup('confirm', {
+                'title': _t('Confirm?'),
+                'body': _t('Execute Fiscal Closure (Report Z) and Daily Financial Report (Report X)'),
+                confirm: function() {
+                    self.chrome.loading_message(_t('Connecting to the fiscal printer'));
+                    fp90.printFiscalReport();
+                    fp90.printFiscalXReport();
+                    self.chrome.loading_hide();
```

Comment:
L'unico problema di questa implementazione forse è che, dopo aver fatto click su "confirm", mentre si attende che la stampante risponda, lo schermo principale ritorna cliccabile.
@monen17 puoi valutare un'implementazione analoga a quella che lancia `printFiscalReport`?

Reply:
Grazie della review!
Credo succeda perché il popup viene chiuso prima di eseguire il confirm in https://github.com/odoo/odoo/blob/5ab778227dfbc93b2ff415ebb0e40ce35953f13e/addons/point_of_sale/static/src/js/popups.js#L78.
Ho lasciato il messaggio di connessione all'interno della conferma per chiarire che da quel momento ci si sta connettendo alla stampante.

Ho aggiunto il loader anche fuori dal popup come succede anche nelle altre funzioni, puoi controllare se ora va meglio?